contracts/0.8.25/vaults/Dashboard.sol

```diff
@@ -159,6 +233,15 @@ contract Dashboard is AccessControlEnumerable {
         _withdraw(_recipient, _ether);
     }
 
+    /**
+     * @notice Withdraws stETH tokens from the staking vault to wrapped ether. Approvals for the passed amounts should be done before.
```

Comment:
Sorry, didn't get what it should do. RN, ether is still on the dashboard contract. Should it be transferred to msg.sender or some other recepient?

Reply:
Updated